index.html

```diff
@@ -0,0 +1,168 @@
+        </div>
+
+        <div class="intro_parag">BEIRUT ART FAIR is the space for artistic revelations in the region.
+             A major event in the art fair calendar, the fair attracts a growing number of collectors, institutions,
+             art professionals and visitors from around the world, totalling 36,000 entries in 2019.
+        </div>
+
+        <div class="daate">
+          <h2><i class="far fa-calendar-alt"></i> 221.9.20(MON) Till 28(TUE)</h2>
+          <h3><i class="fas fa-map-marker-alt"></i> Seaside Arena halls 1 & 2, Beirut New Waterfront, Beirut, Lebanon</h3>
+        </div>
+      </div>
+    </section>
+
+    
+    <section id="project_schedule">
+        <div class="main-program">
+          <h3>Main Program</h3>
+          <hr>
+        </div>
+  
+        <div class="art_programs">
+          <div class="program">
+            <i class="fa fa-paint-brush fa-2x"></i>
+            <h4>Meet the art</h4>
+            <p>LEBANON MODERN!  Unexpected Trove - The Unseen Works of Hussein Madi (Rome 1964-1970)n</p>
+          </div>
+          <div class="program program_2">
+            <i class="fas fa-child fa-2x"></i>
+            <h4>KIDS WORKSHOPS</h4>
+            <p>Enjoy art with your family at the fair! Dedicated tours will be available for children aged 6–12 Given by education professionals</p>
+          </div>
+          <div class="program program_2">
+            <i class="fas fa-microphone-alt"></i>
+            <h4>TALKS AND ROUNTABLES</h4>
+            <p>Our program of roundtables and talks aims at opening up the discussion of the past
+                 and future circumstances of various MENA art scenes in constant evolution.</p>
+          </div>
+
+  
+        </div>
+  
+        <div class="program_btn">
+          <a href="#">See the whole program</a>
+        </div>
+
+      </section>
+
+    <section id="art_guests">
+      <div class="main-program">
+        <h3>Guests</h3>
+        <hr>
+      </div>
+
+      
+      <div class="guest_list"></div>
+
+    </section>
+
+    <section id="art_partners">
+      <div class="main-program">
+        <h3>Partners</h3>
+        <hr>
+      </div>
+
+      <div class="partners">
+        <img src="./Imgs/mozilla.png" alt="Mozilla">
+        <img src="./Imgs/google.png" alt="Google">
+        <img src="./Imgs/naver.png" alt="Naver">
+        <img src="./Imgs/daumkaka.png" alt="Kaka">
+        <img src="./Imgs/airbnb.png" alt="Airbnb">
+      </div>
+    </section>
+  </main>
+
+
+      <footer>
+        <div class="footer_copy">
+          <img src="./Imgs/logo.png" alt="Logo">
+          <aside>
+            <h6>Beirut Art Fair <br>Beirut New Waterfront</h6>
+          </aside>
+        </div>
+    
+        <div class="final-parag">
+          <p>2021 Beirut Art Fair</p>
+          <p>Copy Rights Reserved</p>
+        </div>
+    
+        <div class="go_up">
+          <a href="#"><i class="fas fa-arrow-up"></i></a>
+        </div>
+      </footer>
```

Comment:
The **background color** of your footer on both the mobile and desktop versions are the same. As specified let's ensure to improve upon these aspects:
- [ ]  The footer, on mobile size should have a white background, which is implemented.
- [ ] The footer, on desktop size should have a dark background.

Reply:
- [ ] The footer here should be different from the partner's section. Try and change the background color of the footer to be less dark so that it can differ from the partner's section.